recipes/neuraxle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "neuraxle" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ece9e7b833037554f222d6360cb4317230f6957e042bb2383445a892e7b7ac11
+
+build:
```

Comment:
This might be a candidate for `noarch: python` see:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python